_i18n/en.yml

```diff
@@ -608,24 +610,40 @@ help_pages:
-      #### Enter your email and confirm it
+      #### How do you create an account with login.gov?
+      <img src="site.baseurl/assets/img/help/1-login-gov-welcome.png" class="mt3 mb3 block" alt="Login.gov create account page help image" />
+      To create a login.gov account you’ll need a valid email address and a working phone number. We’ll also ask you to create a password.
 
-      Select “create an account” and enter your email address where indicated. Then check your email account for a message from us. Within the email is a link. Clicking the link will send you back to login.gov with your email confirmed.
-      #### Enter your password
+      Please be prepared to save your password as well as the personal key that is generated for you in case you ever need to recover your account.
 
+      #### Enter your email and confirm it
+      <img src="site.baseurl/assets/img/help/2-login-gov-enteremail.png" class="mt3 mb3 block" alt="Enter your email help image" />
+      Select [“create an account”](https://secure.login.gov/sign_up/enter_email) and enter your email address. Then check your email account for a message from us. Within the email is a link. Clicking the link will confirm your email address and send you back to login.gov.
+      This is what it looks like:
+      <img src="site.baseurl/assets/img/help/3-confirm-your-email.png" class="mt3 mb3 block" alt="Confirm your email help image" />
+
+      #### Create a password
       Next, you will create a strong password. Use the password strength meter on the screen as a guide. Passwords must be at least 8 characters, but otherwise there are no restrictions. You can even enter spaces in between words to get to 8 characters.
-      #### Enter your phone number and confirm it
+      <img src="site.baseurl/assets/img/help/4-choose-a-password.png" class="mt3 mb3 block" alt="Choose password help image" />
 
+      #### Enter your phone number and confirm it
       Next, you will enter a phone number at which you can get calls or SMS messages. If you have a landline, please ask for a security code by a phone call. We will send a unique security code to that phone number each time you sign into your account. This is called two-factor authentication. The second step keeps your account more secure than using only a password.
+      <img src="site.baseurl/assets/img/help/5-phonenumber.png" class="mt3 mb3 block" alt="Add a phone number help image" />
 
-      Each security code expires after 5 minutes and can only be used once. If you don't enter the security code within 5 minutes, just request a new code. Each code is only good once, so no one can steal one you've already used.
+      Each security code expires after 10 minutes and can only be used once. If you don’t enter the security code within 5 minutes, just request a new code. Each code is only good once, so no one can steal one you’ve already used.
```

Comment:
It seems like these errors I keep finding are being undone, or somehow are not fixed? `5 minutes` in the second sentence should be `10 minutes`.

Reply:
Yargh! So frustrating. I'm gonna pull this deploy and fix again.